rotkehlchen/tests/unit/decoders/test_ens.py

```diff
@@ -80,3 +81,210 @@ def test_mint_ens_name(database, ethereum_manager, function_scope_messages_aggre
             'token_name': 'ERC721 token',
         },
     )
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0x4bBa290826C253BD854121346c370a9886d1bC26']])  # noqa: E501
+def test_text_changed(evm_transaction_decoder):
+    """
+    Data taken from
+    https://etherscan.io/tx/0xaa59cb2029651d2ed2c0d1ee34b9b88f0b90278fc6da5b51446d4abf24d7f598
+    """
+    # TODO: For faster tests hard-code the transaction and the logs here so no remote query needed
+    tx_hash = deserialize_evm_tx_hash('0xaa59cb2029651d2ed2c0d1ee34b9b88f0b90278fc6da5b51446d4abf24d7f598')  # noqa: E501
+    events = evm_transaction_decoder.decode_transaction_hashes(ignore_cache=True, tx_hashes=[tx_hash])  # noqa: E501
+    expected_events = [
+        HistoryBaseEntry(
+            event_identifier=HexBytes(
+                '0xaa59cb2029651d2ed2c0d1ee34b9b88f0b90278fc6da5b51446d4abf24d7f598',
+            ),
```

Comment:
Is there any problem vs just using

```suggestion
            event_identifier=tx_hash,
```

?

Reply:
Yeah, no problem indeed